k8s-start.sh

```diff
@@ -6,6 +6,10 @@ sleep 15
 
 NS=parking-snitch-production
 
-kubectl apply -n ${NS} -f inspector-service/inspector-service.yml 
-kubectl apply -n ${NS} -f notification-service/notification-service.yml 
+kubectl apply -n ${NS} -f photo-queue/photo-queue.yml
```

Comment:
what about `photo-service/photo-queue.yml` ?

Reply:
Queue is used by more then one service so it should not belong to any of them, that's why I put it in separate folder